src/app/ApplicationTemplate.Shared.Views/Framework/Connectivity/Connectivity.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using Uno.Extensions;
+using Uno.Logging;
+using Windows.Networking.Connectivity;
+
+namespace ApplicationTemplate;
+
+public sealed class Connectivity : IConnectivity
+{
+	public Connectivity()
+	{
+		NetworkInformation.NetworkStatusChanged += OnNetworkStatusChanged;
+	}
+
+	~Connectivity()
```

Comment:
I would propose to implement `IDisposable` to unsubscribe to follow the dotnet disposable pattern.

Reply:
Good idea.